apps/src/templates/levelSummary/ResponseMenuDropdown.jsx

```diff
@@ -24,21 +62,15 @@ const ResponseMenuDropdown = ({response, hideResponse}) => {
         color={buttonColors.purple}
         size="xs"
         type="tertiary"
-        className={styles.studentAnswerMenuButton}
+        className={classNames(
+          styles.studentAnswerMenuButton,
+          unpinResponse && styles.studentAnswerMenuButtonPinned
+        )}
       />
       {isOpen && (
         <div className={styles.studentAnswerMenuDropdown}>
           <ul>
-            <li>
-              <button
-                type="button"
-                className={styles.dropdownOption}
-                onClick={() => setIsOpen(false)}
-              >
-                <FontAwesomeV6Icon iconName="thumbtack" />
-                {i18n.pinResponse()}
-              </button>
-            </li>
+            <li>{getPinnedDropdownOption()}</li>
             <li>
               <button
```

Comment:
Again, maybe more of a style question, but do we think this would be easier to read if we moved it to a function?  Or does it feel unnecessary?